src/Sylius/Behat/Context/Ui/UserContext.php

```diff
@@ -40,4 +69,47 @@ public function iLogInAs($login, $password)
         $this->loginPage->open();
         $this->loginPage->logIn($login, $password);
     }
+
+    /**
+     * @When I delete the account of :email user
+     */
+    public function iDeleteAccount($email)
+    {
+        $user = $this->userRepository->findOneByEmail($email);
+
+        $this->sharedStorage->set('deleted_user', $user);
+
+        $this->customerShowPage->open(['id' => $user->getCustomer()->getId()]);
+        $this->customerShowPage->deleteAccount();
+    }
+
+    /**
+     * @When I try to delete my own account
+     */
+    public function iTryDeletingMyOwnAccount()
+    {
+        $admin = $this->sharedStorage->get('admin');
+
+        $this->customerShowPage->open(['id' => $admin->getId()]);
```

Comment:
We should reconsider this scenario later, because it feels weird when action "try" does not do anything in the end. :D But I am fine with changing that somewhere down the road.